app/shared/productTypes.tsx

```diff
@@ -351,6 +353,7 @@ export const PRODUCT_TYPES: { [productKey in ProductTypeKeys]: ProductType } = {
     allProductsProductTypeFilterString: "Paper",
     urlPart: "paper",
     getOphanProductType: () => "PRINT_SUBSCRIPTION",
+    showRestrictedNewsletters: ["6009"],
```

Comment:
6009 is the identifier of the voucher newsletter? Would it benefit from being extracted to a value?